src/components/snippets/kubernetesCode.js

```diff
@@ -1,10 +1,12 @@
 export const yamlDbServerCode = (version) => {
   if (version !== 'latest') {
     return `minikube start --memory=8192 --cpus=4 --disk-size=40g --vm-driver=virtualbox
-curl  https://raw.githubusercontent.com/yugabyte/yugabyte-db/master/cloud/kubernetes/yugabyte-statefulset-rf-1.yaml | sed 's/image: yugabytedb\\/yugabyte\\:latest/image: yugabytedb\\/yugabyte:${version}/g' | kubectl  apply -f -`
+kubectl create namespace yb-demo
+curl  https://raw.githubusercontent.com/yugabyte/yugabyte-db/master/cloud/kubernetes/yugabyte-statefulset-rf-1.yaml | sed 's/image: yugabytedb\\/yugabyte\\:latest/image: yugabytedb\\/yugabyte:${version}/g' | kubectl  apply --namespace='yb-demo' -f -`
```

Comment:
```suggestion
    kubectl create namespace yb-demo
    curl  https://raw.githubusercontent.com/yugabyte/yugabyte-db/master/cloud/kubernetes/yugabyte-statefulset-rf-1.yaml | sed 's/image: yugabytedb\\/yugabyte\\:latest/image: yugabytedb\\/yugabyte:${version}/g' | kubectl  apply --namespace='yb-demo' -f -`
```

Reply:
Will this still work correctly with indentation? It's jarring to see without :)

Reply:
@polarweasel It will work, but the copied text will have the indentation as well.

Reply:
It looked fine on the live preview.